### ISSUE TITLE
Accessibility: replace screenshot of table with HTML table

### ISSUE THREAD

**Issue by tobyhodges** [datacarpentry/spreadsheets-socialsci#151]
The table evaluating different choices of null values in [the _Using problematic null values_ section of the _Formatting Problems_ episode](https://datacarpentry.org/spreadsheets-socialsci/02-common-mistakes/index.html#-using-problematic-null-values) is currently a screenshot from the PDF of [the paper it was originally published in](https://ojs.library.queensu.ca/index.php/IEE/article/view/4608). 

This format is not accessible to people accessing the lesson with a screen reader, and it is very difficult to capture all the information in the table as alt-text to the image. I believe the screenshot should be replaced with an HTML table containing the same information.

Note that there is [an identical issue in the _Data Organization in Spreadsheets for Ecologists_ lesson repository](https://github.com/datacarpentry/spreadsheet-ecology-lesson/issues/319), and another in [the Library Carpentry _Tidy Data for Librarians_ lesson repository](https://github.com/LibraryCarpentry/lc-spreadsheets/issues/124), which should be addressed simultaneously. The Library Carpentry repository issue also includes some suggestions for how the descriptions in the table could be improved when converting the screenshot.

**Comment by hoytpr** [comment]
Here is an HTML table of the screenshot submitted for [the spreadsheet ecology lesson#319](https://github.com/datacarpentry/spreadsheet-ecology-lesson/issues/319) .

[null_values_table.txt](https://github.com/datacarpentry/spreadsheets-socialsci/files/10126679/null_values_table.txt)
